### ISSUE TITLE
Cloud Internet Services creation causes crash in IBM Provider

### ISSUE THREAD

**Issue by stevestrutt** [IBM-Cloud/terraform-provider-ibm#376]
### Terraform Version
0.11.8,   IBM Provider 0.11.3

### Affected Resource(s)
- ibm_resource_instance

### Terraform Configuration Files
```hcl
resource "ibm_resource_instance" "internet_srv_instance" {
  name              = "Testing"
  service           = "internet-svcs"
  plan              = "paid"
  location          = "global"

}
```

### Panic Output
[crash.log](https://github.com/IBM-Cloud/terraform-provider-ibm/files/2424281/crash.log)

### Expected Behavior
CIS Service created

### Actual Behavior
Error: Error applying plan:
1 error(s) occurred:
* ibm_resource_instance.internet_srv_instance: 1 error(s) occurred:
* ibm_resource_instance.internet_srv_instance: unexpected EOF
panic: runtime error: invalid memory address or nil pointer dereference
2018-09-27T14:28:02.968+0100 [DEBUG] plugin.terraform-provider-ibm_v0.11.3: [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x1b44516]
2018-09-27T14:28:02.968+0100 [DEBUG] plugin.terraform-provider-ibm_v0.11.3: 
2018-09-27T14:28:02.968+0100 [DEBUG] plugin.terraform-provider-ibm_v0.11.3: goroutine 90 [running]:
2018-09-27T14:28:02.968+0100 [DEBUG] plugin.terraform-provider-ibm_v0.11.3: github.com/terraform-providers/terraform-provider-ibm/ibm.resourceIBMResourceInstanceCreate(0xc4202982a0, 0x1ea7b40, 0xc4201b4000, 0xc4202982a0, 0x0)

### Steps to Reproduce
`terraform apply`

**Comment by hkantare** [comment]
Did you exported BM_API_key or provided bluemix API key ...In tried the same config it fails with different error

<img width="580" alt="image" src="https://user-images.githubusercontent.com/24930306/46154318-89571700-c292-11e8-92ec-e0f3939cd7d4.png">


Because plan "paid" is not a valid argument...Valid plans are `standard` and `trial`

**Comment by hkantare** [comment]
resource_group_id is optional if not provided it creates in default resource group...

**Comment by hkantare** [comment]
Fix is available as part of v0.12.0 .Can you please test with latest provider https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v0.12.0
